telebot/__init__.py

```diff
@@ -5924,13 +5927,14 @@ def _run_middlewares_and_handler(self, message, handlers, middlewares, update_ty
                     if not process_handler: continue
                     for i in inspect.signature(handler['function']).parameters:
                         params.append(i)
+                    ret = None
```

Comment:
Renaming this variable to result would be more appropriate
```suggestion
                    result = None
```
And other variables should be renamed as well.